examples/0_basic/0_create_did.rs

```diff
@@ -56,10 +58,16 @@ async fn main() -> anyhow::Result<()> {
   let mut document: IotaDocument = IotaDocument::new(&network_name);
 
   // Insert a new Ed25519 verification method in the DID document.
-  let keypair: KeyPair = KeyPair::new(KeyType::Ed25519)?;
-  let method: VerificationMethod =
-    VerificationMethod::new(document.id().clone(), keypair.type_(), keypair.public(), "#key-1")?;
-  document.insert_method(method, MethodScope::VerificationMethod)?;
+  let storage: MemStorage = MemStorage::new(JwkMemStore::new(), KeyIdMemstore::new());
+  let _fragment: String = document
```

Comment:
```suggestion
  document
```

I would just ignore the return value, so it's a bit cleaner.